build/Makefile

```diff
@@ -856,3 +856,10 @@ test-gen-crd-client:
 		echo "$$diff_output"; \
 	fi
 
+update-agones: $(ensure-build-image)
+	docker run -it --rm \
+		$(common_mounts) \
+		-w $(workdir_path) \
+		-e "KUBECONFIG=/root/.kube/$(kubeconfig_file)" \
```

Comment:
```suggestion
```

Is this line required? I feel like it should be where it looks by default 🤔

Reply:
excluded this line